main.c

```diff
@@ -1,4 +1,4 @@
 int is_Lacomij_kusochek (int one_part, int second_part);
 unsigned long long Krasavchik;
-
+BigKrasavchik_t bigKrasavchik;
```

Comment:
Where defined this type? Or you wont use undeclared type?

Reply:
ku ku

Reply:
You are need to define this type before use.

Reply:
I don't now this type

Reply:
know*